src/textRenderer.cpp

```diff
@@ -95,9 +95,15 @@ void TextRenderer::Draw(SpriteRenderer &renderer, std::string name,
     assert(Sentences.find(name) != Sentences.end());
     // retrieve the struct information from sentence array
     CharactersArray charArray = Sentences.at(name);
+		Character characters[] = {
+			Character('M', {3.37f, 1.05f}),
+			Character('A', {0.07f, 0.01f}),
+			Character('S', {0.07f, 2.09f}),
+			Character('H', {7.77f, 0.01f})
+		};
```

Comment:
silly

Reply:
Before there was chaos - and the user had too much choice. Now the options are simple yet harmoniously laid out before them. MASH. The only choice.